transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -26,6 +26,9 @@
 
     public static final ChannelOption<Boolean> SO_REUSEPORT = valueOf("SO_REUSEPORT");
 
+    public static final ChannelOption<EpollDomainSocketReadMode> DOMAIN_SOCKET_READ_MODE =
```

Comment:
Perhaps we should move this to `ChannelOption` and rename `EpollDomainSocketReadmode`? .. because transferring a file descriptor doesn't seem Linux-only.